integration/common/core.py

```diff
@@ -372,10 +373,10 @@ def start_no_frontend_volume(grpc_c, *grpc_r_list):
     grpc_c.volume_frontend_start(FRONTEND_TGT_BLOCKDEV)
 
     for grpc_r in grpc_r_list:
-        open_replica(grpc_r)
+        open_replica(grpc_r, SIZE_STR)
```

Comment:
I found `def open_replica(grpc_client, backing_file=None)` did not change and the second arg is `backing_file`.

Is this correct?

Reply:
Ah, forgot updating one of the two open_replica().

Reply:
Updated. Thank you.